src/demos/Badges.vue

```diff
@@ -20,10 +20,10 @@ components: {
         <h2>Standard Badges</h2>
         <DemoSection title="Standard Badges" :code="demoCode">
           <template #components>
+            <Badge variant="beta" :align-left="true">Beta</Badge>
             <Badge variant="new" :align-left="true">New</Badge>
+            <Badge variant="warning" :align-left="true">Error</Badge>
             <Badge variant="tag" :align-left="true">Tag</Badge>
```

Comment:
Is `:align-left="true"` not default ? It seems like all of the badges are set as `:align-left="true"`

Reply:
There are a lot of cases when you don't need it to align left. For example, when used on left nav in sl.com.

Reply:
ok! get it!